src/openepd/model/epd.py

```diff
@@ -30,6 +30,9 @@
 from openepd.model.specs import Specs
 from openepd.model.standard import Standard
 
+OPENEPD_VERSION = "0.1"
+"""Current implemented version of openEPD data format."""
+
 
 class Epd(WithAttachmentsMixin, WithAltIdsMixin, BaseOpenEpdSchema):
```

Comment:
Maybe we should declare this as `class EpdV0`

And add an alias like `Epd = EpdV0`?

Reply:
Moreover, we might need to create a factory or factory method/function which allows you to build Epd object from given json/dict.

It should check the version first, and create corresponding class instance.

Reply:
added.